docker-compose.yml

```diff
@@ -26,7 +26,6 @@ services:
       context: .
       dockerfile: Dockerfile.dev
     image: panelapp_web
-    restart: on-failure
```

Comment:
could you revert this change? It should be restarted if it fails.
We need to sort out tests + random commands runs, but it's a separate issue